package.json

```diff
@@ -35,64 +32,65 @@
   },
   "homepage": "https://github.com/Ivan-Marquez/momentum#readme",
   "standard": {
+    "globals": [
+      "sinon",
+      "expect",
+      "describe",
+      "context",
+      "it",
+      "beforeEach",
+      "assert",
+      "clearModule",
+      "config"
+    ],
     "ignore": [
-      "public/"
+      "src/config/mongoose_seed/"
```

Comment:
Now that I remember, `standard` doesn't like the files inside the `json_generator` folder inside `mongoose_seed`. I think it's because of how the `JSON` structure is written in these files. We could add `mongoose_seed`, but not the generators.